### PR TITLE
PlanetScale -> GoogleCloud Migration

### PR DESCRIPTION
**Pull request by shn-srl**
:heavy_multiplication_x: : Remove Prisma dependency and all references to it -> Will do in follow up ticket

:heavy_check_mark: Remove previous api routes / calls

**Transform ExerciseSubmission Prisma/PlanetScale calls to GraphQL/GoogleCloud calls **

- Mutation is located in ```apps/web/src/mutations/use-experiment-create-exercise-submission-mutation.ts```

:heavy_check_mark: Replace ExerciseSubmission (includes SpoilerOpened)
:heavy_check_mark: Remove AbSubmission (AB Testing data)
:heavy_check_mark: Remove QuickbarStatsSubmission
:heavy_check_mark: Remove EquationsAppTracking

### CODE REVIEW

**Comment by elbotho** [comment]
Do you know where the Bundle size increase comes from?

**Comment by shn-srl** [comment]
> Do you know where the Bundle size increase comes from?

I see a decrease :sweat_smile: 

image


Not sure atm. Will check it out. Will also remove prisma totally so that should be a reduction too

**Comment by Entkenntnis** [comment]
This is the important bundle size number:

grafik

The increase is a result of graphql-request being loaded in the frontend. We can currently work without it being loaded. I would recommand to lazy load it dynamically (as the feature is not crucial for the page to work)

File: apps/web/src/helper/exercise-submission.ts [hunk 1]
```diff
@@ -14,41 +13,40 @@ export interface ExerciseSubmissionData {
...
 
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-      const trigger = (window as any)?.__triggerRender
-      if (typeof trigger === 'function') {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-        trigger()
-      }
-    }
+  if (ab?.experiment === 'dreisatz_new_design') {
+    handleDreisatzNewDesign(data, entityId)
```
Comment (shn-srl) [inline apps/web/src/helper/exercise-submission.ts:49]: Can this be removed?

**Review by Entkenntnis — APPROVED** [review]

**Pull request merged**